addons/web/static/tests/_framework/view_test_helpers.js

```diff
@@ -227,7 +227,7 @@ export async function mountView(params, target = null) {
     after(() => actionManagerEl.remove());
     return mountWithCleanup(View, {
         env: params.env,
-        componentEnv: { config: params.config },
+        componentEnv: { config: { actionCache: true, ...params.config } },
```

Comment:
in view.js, add `cache: true` to the default config (also rename actionCache into cache)